examples/routing_service/routing_service_msql_adapter/README.md

```diff
@@ -0,0 +1,27 @@
+# Example Code: Routing Service Microsoft MSQL Adapter
+
+## Concept
+
+Adapters are pluggable components that allow RTI Routing Service to consume and
+produce data for different data domains (e.g., DDS, JMS, Socket, etc.).
+
+By default, RTI Routing Service is distributed with a built-in DDS adapter. Any
+other adapter plugins must be provided as a set of C, C++11 libraries or Java
+classes registered within the `<adapter_library>` tag.
+
+Input adapters are used to collect data samples from different data domains,
+such as Connext DDS or JMS. The input samples are processed by the Routing Service
+engine and are passed along to custom transformations if they are defined.
+Finally, the transformed data is provided to the output adapters. The adapter
+plugin API is used to create new adapters; it is supported in C, C++11 and Java.
+
+## Example Description
```

Comment:
In the example description, I would make it very clear that this example works (as far as I have understood) only with Shapes Demo topics. That is the case, right?

Reply:
done at the toplevel Readme.md